### PR TITLE
Try the monitor command

### PR DESCRIPTION
**Pull request by jeffalder**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -32,6 +32,8 @@ dependencies {
     api("io.micrometer:micrometer-core:1.3.3")
     api("com.newrelic.telemetry:telemetry:0.6.1")
     implementation("org.slf4j:slf4j-api:1.7.30")
+    implementation("ch.qos.logback:logback-classic:1.2.3")
+    implementation("ch.qos.logback:logback-core:1.2.3")
```
Comment (breedx-nr) [inline build.gradle.kts:36]: Why?  Last I remembered, logback is dual licensed with LGPL included.  Do we really want/need to do this?

File: .github/workflows/snyk.yml [hunk 2]
```diff
@@ -10,4 +10,4 @@ jobs:
         env:
           SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
         with:
-          command: test
\ No newline at end of file
+          command: monitor
```
Comment (breedx-nr) [inline .github/workflows/snyk.yml:13]: Is this just a test PR or are you looking to go forward with this technique if it works?  If so, can you describe what this changes please?

File: build.gradle.kts (hunk 1 above)
Comment (jeffalder) [reply build.gradle.kts:36]: This is a snyk test. I'm trying to figure out why it's failing even though the issue is ignored in Snyk.

File: .github/workflows/snyk.yml (hunk 2 above)
Comment (jeffalder) [reply .github/workflows/snyk.yml:13]: Test pr

File: build.gradle.kts (hunk 1 above)
Comment (jeffalder) [reply build.gradle.kts:36]: I want to see the Snyk check fail _with_ these libs and pass _without_ them.

**Pull request closed without merging**